instrumentation/rxjava-2.0/javaagent/src/main/java/io/opentelemetry/instrumentation/rxjava2/RxJava2InstrumentationModule.java

```diff
@@ -18,6 +19,7 @@
 import net.bytebuddy.description.type.TypeDescription;
 import net.bytebuddy.matcher.ElementMatcher;
 
+@AutoService(InstrumentationModule.class)
```

Comment:
um, how were tests passing? 😟

Reply:
ah, looks like rxjava2 javaagent tests just need the executor instrumentation to pass, I guess that's not too surprising

Reply:
It should be. This means that test is not sensitive and thus useless :(

Reply:
cc: @piotr-sumo when you have time can you see if there's a test we can add for rxjava2 that doesn't pass purely due to executor instrumentation?

Reply:
This is being addressed in #2672